ocp_install_env.sh

```diff
@@ -312,8 +315,13 @@ spec:
   bootMACAddress: $mac
   bmc:
     address: $address
-    credentialsName: ${name}-bmc-secret
-EOF
+    credentialsName: ${name}-bmc-secret"
 
+        echo "${secret}${bmh}" >> "${outdir}/${host_output}"
+
+        # Extra files will be used later to generate a secret used by e2e tests
```

Comment:
Maybe we can do this only if `OPENSHIFT_CI` is set?. Because it looks like apart from CI, this will not be used. For local or any other runs, extra-hosts yaml file is sufficient.

Reply:
In reality I guess it could be useful also for normal development: sometimes I allocate more extra workers, and deploy them one-by-one or in batch, so I ended up cutting&paste the global file